src/main/resources/assets/viafabric/lang/de_de.json

```diff
@@ -3,5 +3,12 @@
   "gui.enable_client_side_button": "ViaVersion aktivieren",
   "gui.enable_client_side.question": "Bist du dir sicher, dass du den clientseitigen Modus aktivieren möchtest?",
   "gui.enable_client_side.warning": "Ich kann nicht garantieren, dass diese Mod auf jedem (oder überhaupt einem) Server erlaubt ist. Diese Mod könnte eventuell Probleme mit Anti-Cheat Plugins hervorrufen. VERWENDUNG AUF EIGENE GEFAHR.",
-  "gui.enable_client_side.enable": "Aktivieren"
+  "gui.enable_client_side.enable": "Aktivieren",
+  "gui.viafabric_config.title": "ViaFabric-Konfigurationen",
+  "gui.client_side.enable": "Client-seitig aktivieren",
```

Comment:
This should be _Clientseitigen Modus aktivieren_

Reply:
yes you added mod to the traduction but in the original text there wasn't the word mod